### PR TITLE
no need to prepend dependency name to path when globing build scope

### PR DESCRIPTION
**Pull request by mingwli**
[DevOps#124841](https://ceapex.visualstudio.com/Engineering/_workitems/edit/124841)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5146)

### CODE REVIEW

File: src/docfx/lib/path/FilePath.cs [hunk 1]
```diff
@@ -104,7 +104,7 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            return HashCode.Combine(PathUtility.PathComparer.GetHashCode(Path), Origin, DependencyName, Commit);
```
Comment (OsmondJiang) [inline src/docfx/lib/path/FilePath.cs]: do we need change compare?

File: src/docfx/build/context/BuildScope.cs [hunk 2]
```diff
@@ -95,8 +95,7 @@ public bool GetActualFileName(string fileName, out string actualFileName)
 
                 ParallelUtility.ForEach(fileNames, file =>
                 {
-                    var path = Path.Combine(dependencyName ?? "", file.Path).Replace("\\", "/");
```
Comment (OsmondJiang) [inline src/docfx/build/context/BuildScope.cs]: ooooops, my mistake

Comment (OsmondJiang) [reply src/docfx/build/context/BuildScope.cs]: forget to delete

**Comment by mingwli** [comment]
> Is there a test to check if metadata are applied correctly to files in CRR repos using the correct path?

was added in this pr [#5142](https://github.com/dotnet/docfx/pull/5142/files#diff-f5af9eb28100acb0f4e35f1a92fafc22R354)

**Pull request merged**